datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -170,7 +171,16 @@ def __init__(self, instance, init_config, remapper=None, logger=None):
         auth = None
         if config['password']:
             if config['username']:
-                auth = (config['username'], config['password'])
+                if config['auth_type']:
+                    if config['auth_type'] == 'basic':
```

Comment:
Probably better to convert to lowercase before so that we accept `auth_type: Digest`